docs/CHANGELOG.asciidoc

```diff
@@ -37,6 +38,274 @@ This section summarizes the changes in each release.
+* Fixes anomaly dot plotted in wrong location in Single Metric Viewer {pull}66071[#66071]
+Management::
+* Fixes error when deleting snapshots behind reverse proxy {pull}66147[#66147]
+* Fixes bug when editing retention of slm policy {pull}67137[#67137]
+* Fixes fetch policies query {pull}67827[#67827]
+* Adds missing time unit labels {pull}68205[#68205]
+* Fixes pagination bugs in CCR and Remote Clusters {pull}65931[#65931]
+* Cleans up doc title when navigating away from Snapshot Restore {pull}67906[#67906]
+* Fixes boolean cell values in analytics table result views and transforms wizard {pull}62618[#62618]
+* Index pattern management - field list - localization functions returning empty strings {pull}64055[#64055]
+Maps::
+* Fixes replaceLayerList to handle case where map is not intialized {pull}62202[#62202]
+* Safely handles empty string and invalid strings from EuiColorPicker {pull}62507[#62507]
+* Do not show circle border when symbol size is zero {pull}62644[#62644]
+* Fixes attribution overflow with exit full screen button {pull}62699[#62699]
+* Creates NOT EXISTS filter for tooltip property with no value {pull}62849[#62849]
+* Fixes bug where toggling Scaling type does not re-fetch data {pull}63326[#63326]
+* Fixes double fetch when filter pill is added {pull}63024[#63024]
+* Fixes term join agg key collision {pull}63324[#63324]
+* Fixes date labels {pull}63909[#63909]
+* Removes SLA percentage metric {pull}65718[#65718]
+* Removes dateline check for geo_shape queries, split geo_bounding_box queries that cross dateline into 2 boxes {pull}64598[#64598]
+* Handles case where fit to bounds does not match any documents {pull}66307[#66307]
+* Returns bounding box for static feature collection without joins {pull}66607[#66607]
+* Fixes legend icon color for custom categorial palettes {pull}67141[#67141]
+Metrics::
+* Disables search bar when live stream is on {pull}65491[#65491]
+* Ensures live stream always gets latest entries {pull}67935[#67935]
+* Fixes time picker layout issues on Inventory View {pull}66094[#66094]
+* Fixes validation for threshold values {pull}66281[#66281]
+* Prevents component errors from breaking the whole UI {pull}65456[#65456]
+* Fixes mistake in container ip field name {pull}66198[#66198]
+* Fixes time state bug {pull}67630[#67630]
+* Require filterQuery to be ES JSON {pull}64937[#64937]
+* Debounce onChange in alert creation UI {pull}65167[#65167]
+* Fixes default metric alert interval for new conditions {pull}66610[#66610]
+Platform::
+* Fixes inconsistent search behaviour in Advanced Settings {pull}64510[#64510]
+* Refactors nav links and header components {pull}66685[#66685]
+* SavedObjects bulkCreate API should return migrationVersion and strip the type &amp; namespace from the id {pull}65150[#65150]
+* Fixes task manager cancel warning {pull}63756[#63756]
+* New Kibana app link order {pull}67991[#67991]
+* Warning and link to support matrix for IE11 {pull}66512[#66512]
```

Comment:
Maybe we should move this up to the top section to call this out more as a deprecation? May also want to rename to something like `Deprecates IE11 support`